root/components/Header.js

```diff
@@ -0,0 +1,45 @@
+/*
+ * @flow
+ * Copyright (C) 2018 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+const React = require('react');
+const {l} = require('../static/scripts/common/i18n');
+const EntityTabs = require('./EntityTabs');
+const Frag = require('./Frag');
+const SubHeader = require('./SubHeader');
+const EntityLink = require('../static/scripts/common/components/EntityLink');
+const TaggerIcon = require('../static/scripts/common/components/TaggerIcon');
+
+type Props = {
+  entity: EntityT;
+  editTab?: string | React.Node;
+  headerClass: string;
+  heading?: string | React.Node;
+  page: string;
+  subHeading: string;
+};
+
+const TaggableEntities = ['recording', 'reelase'];
```

Comment:
lowercase 't' in `taggableEntities` (only classes should use PascalCase) and s/reelase/release/